packages/console/src/pages/SignInExperience/constants.ts

```diff
@@ -0,0 +1,3 @@
+import { SignInMethods } from '@logto/schemas';
+
+export const signInMethods: Array<keyof SignInMethods> = ['username', 'sms', 'email', 'social'];
```

Comment:
This seems unnecessary to me. You can directly use `keyof SignInMethods` to get an array of the keys

Reply:
This file is unnecessary, removed.